### PR TITLE
improvements to console / codemirror configuration

### LINKED ISSUE #710: SQL statement autocompletion intervenes on a visual level
Hi there.

Following the tutorial at [1], I invoked queries like

```sql
CREATE USER "Custom User";
```
```sql
GRANT DQL ON SCHEMA doc TO "Custom User";
```

While that works perfectly, the `User` string gets capitalized by the automatic SQL-syntax based formatting like

image

This is somehow confusing as it makes the user believe it would not work. However, it still does, the query is submitted as typed.

With kind regards,
Andreas.

[1] https://crate.io/docs/crate/reference/en/4.4/admin/user-management.html#create-user

### PR DESCRIPTION
**Pull request by proddata**
- Remove min-width for columns in console to reduce scrolling
- Change syntax highlighting in console. Keywords in double quotes are now longer  highlighted. Types are highlighted with a different color.
- Activated codemirror code hints for keywords

closes #710 

## Summary of the changes / Why this is an improvement
image

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by proddata** [comment]
if we want to keep using **codemirror**, it might be a good idea to make a pr to include crate-syntax there

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,10 @@ Unreleased
 
 - Remove all analytics (UDC, Segment) completely
 - Remove "Notifications" section in statusbar
+- Remove min-width for columns in console to reduce scrolling
+- Change syntax highlighting in console. Keywords in double quotes are now longer
+  highlighted. Types are highlighted with a different color.
+- Activated codemirror code hints for keyswords
```
Comment (seut) [inline CHANGES.rst:14]:
```suggestion
- Activate codemirror code hints for keywords
```

**Review by seut — APPROVED** [review]
Lgtm, thx! 👍

**Review by amotl — APPROVED** [review]
Thanks a stack!

**Pull request merged**